easybuild/easyblocks/t/tensorflow.py

```diff
@@ -165,10 +171,22 @@ def configure_step(self):
                 'TF_CUDA_COMPUTE_CAPABILITIES': ','.join(self.cfg['cuda_compute_capabilities']),
                 'TF_CUDA_VERSION': get_software_version('CUDA'),
             })
-        if cudnn_root:
+            if cudnn_root:
+                config_env_vars.update({
+                    'CUDNN_INSTALL_PATH': cudnn_root,
+                    'TF_CUDNN_VERSION': get_software_version('cuDNN'),
+                })
+            else:
+                raise EasyBuildError("TensorFlow has a strict dependency on cuDNN if CUDA is enabled")
+            if nccl_root:
+                nccl_version = get_software_version('NCCL')
+                config_env_vars.update({
+                    'NCCL_INSTALL_PATH': nccl_root,
+                })
+            else:
+                nccl_version = '1.3'  # Use simple downloadable version
```

Comment:
@akesandgren Where did you get this `1.3` from? Isn't this something that will vary across TF versions?

Reply:
1.3 is the fallback version if there is no external NCCL v2.x installed (requires manual download).
1.3 can be auto-downloaded and is the version they mention in all TF versions from 1.4, i think, as the default if no external NCCL is available.

But, from TF 1.10 they have changed the Q&A stuff so one now must specify it.